Changed the support email address

brettansley opened the pull request:
The support email address has changed and this commit is to update it
to the new address in the two places it appears in documentation: on
the main index page and in the page footers.

Commit:
Changed the support email address

The support email address has changed and this commit is to update it
to the new address in the two places it appears in documentation: on
the main index page and in the page footers.

combor commented:
It seems to be mentioned also in this file: https://github.com/alphagov/paas-trial-documentation/blob/16d9afdfcb414580a66328477c75ebd13ad1bc38/CloudFoundry/content/apps/quotas.md

Commit:
Updated support email in Quotas file

Updated the support email in the Quotas file to the new one.

brettansley commented:
Thanks - I've updated that one too - the formatting was wrong on the quotas
file too so I've also updated the format so it recognises the email as an
email instead of a URI.

On Thu, Jan 21, 2016 at 4:49 PM, Piotr Komborski <notifications@github.com>
wrote:

> It seems to be mentioned also in this file:
> https://github.com/alphagov/paas-trial-documentation/blob/16d9afdfcb414580a66328477c75ebd13ad1bc38/CloudFoundry/content/apps/quotas.md
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/paas-trial-documentation/pull/13#issuecomment-173631316>
> .
>
